### ISSUE TITLE
Possible filter to show domains blocked by on-device blocklists

### ISSUE THREAD

**Issue by hussainmohd-a** [celzero/rethink-app#1207]
A user writes
> Do you think it would be possible to add a visual log of websites "Blocked" by the apps on-device DNS Blocklist? And also a feature to modify the on-device blocklists to add certain websites to an Allow-list or Deny-list?

**Comment by RohitSurwase** [comment]
Yes, this feature would be useful to quickly find and unblock certain domains.
